tools/stack_decode.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# Call addr2line as needed to resolve addresses in a stack trace,
+# de-interleaving the log lines from multiple threads if required.
+#
+# Two ways to call:
+#   1) No arguments means this script will treat stdin as log output from
+#   Envoy
+#   2) Some arguments means run a subprocess with that command line and send
+#   stderr through the script
+#
+# In each case this script will decode any backtrace log lines found and echo
+# back all non-Backtrace lines untouched.
+
+import collections
+import re
+import subprocess
+import sys
+
+Backtrace = collections.namedtuple("Backtrace",
+                                   "log_prefix obj_file, address_list")
```

Comment:
No reason, I think it came from my originally using commas and pyformat tool mangling it.  I'll push one more fix for this.